GOVERNANCE.md

```diff
@@ -0,0 +1,152 @@
+
+A vote on a single proposal is considered successful if at least two thirds of those eligible to vote vote in favor.
+
+If there are multiple alternatives, members may vote for one or more alternative, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+## FAQ
+
+This section is informational. In case of disagreement, the rules above overrule any FAQ.
+
+### How do I propose a decision?
+
+Send an email to [the developer mailing list](devs) with your motion. If there is no objection within a reasonable amount of time, consider the decision made. If there are objections and no consensus can be found, a vote may be called by a team member.
+
+### How do I become a team member?
+
+To become an official team member, you should make ongoing contributions to one or more project(s) for at least three months. At that point, a team member (typically a maintainer of the project) may propose you for membership. The discussion about this will be held in private, and you will be informed privately when a decision has been made. A possible, but not required, graduation path is to become a maintainer first.
+
+Should the decision be in favor, your new membership will also be announced on the [developers mailing list][devs].
+
+### How do I add a project?
+
+As a team member, propose the new project on the [developer mailing list][devs]. If nobody objects, create the project in the Github organization. Add at least a README.md explaining the goal of the project, and a MAINTAINERS.md with the maintainers of the project (at this point, this probably means you).
+
+### How do I archive or remove a project?
+
+Send an email to the [developer mailing list][devs] proposing the retirement of a project. If nobody objects, move it to the prometheus-junkyard Github organization.
+
+### How do I remove an inactive maintainer?
+
+A maintainer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year will be treated as if they had resigned. If there is an urgent need to replace a maintainer, discuss this on the [team mailing list][team].
+
+### How do I remove a team member?
+
+Team members may resign by notifying the [team mailing list][team]. If you think a team member should be removed against their will, propose this to the [team mailing list][team]. Discussions will be held there in private.
+
+[team]: https://groups.google.com/forum/#!forum/prometheus-team
+[gh]: https://github.com/prometheus
+[devs]: https://groups.google.com/forum/#!forum/prometheus-developers
+[maintainers.md]: https://github.com/search?l=&q=org%3Aprometheus+filename%3AMAINTAINERS.md&ref=advsearch&type=Code&utf8=%E2%9C%93
```

Comment:
This link that goes straight to the MAINTAINERS.md file might be better here: https://github.com/prometheus/prometheus/blob/c686f40adfe5e023f8af81fd17023714c13f2f80/MAINTAINERS.md

Reply:
The reference is about maintainers in any of the projects in our org, which are spread over the many `MAINTAINERS.md` files (one per project). So I think the search link is more applicable?